### PR TITLE
repin cryptography for openssl security update

### PR DESCRIPTION
**Pull request by alexzorin**
https://www.openssl.org/news/secadv/20230207.txt
https://cryptography.io/en/latest/changelog/#v39-0-1

### CODE REVIEW

File: certbot-ci/certbot_integration_tests/certbot_tests/assertions.py [hunk 1]
```diff
@@ -22,13 +22,13 @@
 ADMINS_SID = 'S-1-5-32-544'
 
 
-def assert_elliptic_key(key: str, curve: Type[EllipticCurve]) -> None:
+def assert_elliptic_key(key_path: str, curve: Type[EllipticCurve]) -> None:
```
Comment (ohemorange) [inline certbot-ci/certbot_integration_tests/certbot_tests/assertions.py:25]: Was changing the variable name necessary to get something to pass, or just for clarity?

**Comment by ohemorange** [comment]
Other than my one question, code looks good! Nice to see more accurate type checking.

File: certbot-ci/certbot_integration_tests/certbot_tests/assertions.py (hunk 1 above)
Comment (alexzorin) [reply certbot-ci/certbot_integration_tests/certbot_tests/assertions.py:25]: There is a local variable in the function body called `key` with a different type, and it was being shadowed by the function argument. I felt that it made more sense to change the function argument.

**Review by ohemorange — APPROVED** [review]
lgtm!

**Pull request merged**